client/state/action-types.js

```diff
@@ -23,6 +23,9 @@ export const FETCH_SITE_PLANS = 'FETCH_SITE_PLANS';
 export const FETCH_SITE_PLANS_COMPLETED = 'FETCH_SITE_PLANS_COMPLETED';
 export const FETCH_WPORG_PLUGIN_DATA = 'FETCH_WPORG_PLUGIN_DATA';
 export const NEW_NOTICE = 'NEW_NOTICE';
+export const POST_REQUEST = 'POST_REQUEST';
+export const POST_REQUEST_SUCCESS = 'POST_REQUEST_SUCCESS';
+export const POST_REQUEST_FAILURE = 'POST_REQUEST_FAILURE';
```

Comment:
>I wonder if we should flip all of these to keymirror

What advantage would there be in using keyMirror?

For me at least, this is less about feeling of security with `const` and rather in opting to use simple language constructs than an external library that, in my opinion, doesn't provide much value.

There's some discussion in the `keyMirror` repository [about minifiers crushing keys](https://github.com/STRML/keyMirror#why), though I don't think it affects our situation.